includes/rest-api/class-amp-rest-fonts-controller.php

```diff
@@ -0,0 +1,256 @@
+		}
+
+		$context = ! empty( $request['context'] ) ? $request['context'] : 'view';
+		$data    = $this->add_additional_fields_to_object( $data, $request );
+		$data    = $this->filter_response_by_context( $data, $context );
+		// Wrap the data in a response object.
+		$response = rest_ensure_response( $data );
+
+		/**
+		 * Filters a font returned from the API.
+		 *
+		 * Allows modification of the font right before it is returned.
+		 *
+		 * @param WP_REST_Response $response The response object.
+		 * @param Object $font The original comment object.
+		 * @param WP_REST_Request $request Request used to generate the response.
+		 */
+		return apply_filters( 'rest_prepare_font', $response, $font, $request );
+	}
+
+
+	/**
+	 * Checks if a given request has access to get fonts.
+	 *
+	 * @param WP_REST_Request $request Full data about the request.
+	 *
+	 * @return true|WP_Error True if the request has read access, WP_Error object otherwise.
+	 */
+	public function get_items_permissions_check( $request ) {
+		return current_user_can( 'edit_posts' );
+	}
+
+	/**
+	 * Checks if the font can be read.
+	 *
+	 * @param Object $font Font object.
+	 *
+	 * @return bool Whether the comment can be read.
+	 */
+	protected function check_read_permission( $font ) {
+		return current_user_can( 'edit_posts' );
+	}
+
+	/**
+	 * Retrieves the font' schema, conforming to JSON Schema.
+	 *
+	 * @return array Item schema data.
+	 */
+	public function get_item_schema() {
+		if ( $this->schema ) {
+			return $this->add_additional_fields_schema( $this->schema );
+		}
+		$schema       = [
+			'$schema'    => 'http://json-schema.org/draft-04/schema#',
+			'title'      => 'font',
+			'type'       => 'object',
+			'properties' => [
+				'name'      => [
+					'description' => __( 'The title for the status.', 'amp' ),
+					'type'        => 'string',
+					'context'     => [ 'embed', 'view', 'edit' ],
+					'readonly'    => true,
+				],
+				'slug'      => [
+					'description' => __( 'An alphanumeric identifier for the status.', 'amp' ),
+					'type'        => 'string',
+					'context'     => [ 'embed', 'view', 'edit' ],
+					'readonly'    => true,
+				],
+				'handle'    => [
+					'description' => __( 'An alphanumeric identifier for the handle.', 'amp' ),
+					'type'        => 'string',
+					'context'     => [ 'embed', 'view', 'edit' ],
+					'readonly'    => true,
+				],
+				'fallbacks' => [
+					'description' => __( 'Array of fallback fonts', 'amp' ),
+					'type'        => 'array',
+					'context'     => [ 'embed', 'view', 'edit' ],
+					'readonly'    => true,
+					'default'     => [],
+				],
+				'weights'   => [
+					'description' => __( 'Array of fallback fonts', 'amp' ),
+					'type'        => 'array',
+					'context'     => [ 'embed', 'view', 'edit' ],
+					'readonly'    => true,
+					'default'     => [ 'normal', 'bold', 'bolder', 'lighter' ],
+				],
+				'src'       => [
+					'description' => __( 'URL for the object font.', 'amp' ),
```

Comment:
```suggestion
					'description' => __( 'URL to enqueue the font', 'amp' ),
```

Reply:
Fixed in https://github.com/ampproject/amp-wp/pull/3998/commits/060b83b2257028b4e06c37bec91b905513ecbb6f